server/openslides/motions/migrations/0038_motionvote_user_token.py

```diff
@@ -0,0 +1,32 @@
+# Generated by jsangmeister on 2021-03-18 16:27
+
+from django.db import migrations, models
+
+import openslides.poll.models
+
+from ...poll.migrations.vote_migration_helper import set_user_tokens
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("motions", "0037_motionvote_delegated_user"),
+    ]
+
+    operations = [
+        migrations.AddField(
```

Comment:
You need to split the three fields into three migration files. Postgresql cannot mix data and schema changes. Same for assignments